# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1445237751 -7200
#      Mon Oct 19 08:55:51 2015 +0200
# Branch stable
# Node ID 410b1b3d109e4acf02a10cae3e64eb58748c2d98
# Parent  1c4c4bb8be945255c150420608a134fb1c99e23d
plot.m: use actual property names in the description FMT argument (bug #46240)

* plot.m (doc): concistently use "marker" instead of "markerstyle" to describe the FMT argument
* plot.m (doc): concistently use "displayname" instead of "key" to describe the FMT argument

diff --git a/scripts/plot/draw/plot.m b/scripts/plot/draw/plot.m
--- a/scripts/plot/draw/plot.m
+++ b/scripts/plot/draw/plot.m
@@ -94,22 +94,24 @@
 ## the same number of rows and columns and no attempt is made to transpose
 ## the arguments to make the number of rows match.
 ## @end itemize
 ##
 ## Multiple property-value pairs may be specified, but they must appear
 ## in pairs.  These arguments are applied to the line objects drawn by
 ## @code{plot}.  Useful properties to modify are @qcode{"linestyle"},
 ## @qcode{"linewidth"}, @qcode{"color"}, @qcode{"marker"},
-## @qcode{"markersize"}, @qcode{"markeredgecolor"}, @qcode{"markerfacecolor"}.
+## @qcode{"markersize"}, @qcode{"markeredgecolor"}, @qcode{"markerfacecolor"}.  
+## @xref{Line Properties}.  
 ##
 ## The @var{fmt} format argument can also be used to control the plot style.
-## The format is composed of three parts: linestyle, markerstyle, color.
-## When a markerstyle is specified, but no linestyle, only the markers are
-## plotted.  Similarly, if a linestyle is specified, but no markerstyle, then
+## It is a string composed of four optional parts:
+## "<linestyle><marker><color><;displayname;>".
+## When a marker is specified, but no linestyle, only the markers are
+## plotted.  Similarly, if a linestyle is specified, but no marker, then
 ## only lines are drawn.  If both are specified then lines and markers will
 ## be plotted.  If no @var{fmt} and no @var{property}/@var{value} pairs are
 ## given, then the default plot style is solid lines with no markers and the
 ## color determined by the @qcode{"colororder"} property of the current axes.
 ##
 ## Format arguments:
 ##
 ## @table @asis
@@ -117,17 +119,17 @@
 ##
 ## @multitable @columnfractions 0.06 0.94
 ## @item @samp{-}  @tab Use solid lines (default).
 ## @item @samp{--} @tab Use dashed lines.
 ## @item @samp{:}  @tab Use dotted lines.
 ## @item @samp{-.} @tab Use dash-dotted lines.
 ## @end multitable
 ##
-## @item markerstyle
+## @item marker
 ##
 ## @multitable @columnfractions 0.06 0.94
 ## @item @samp{+} @tab crosshair
 ## @item @samp{o} @tab circle
 ## @item @samp{*} @tab star
 ## @item @samp{.} @tab point
 ## @item @samp{x} @tab cross
 ## @item @samp{s} @tab square
@@ -147,21 +149,21 @@
 ## @item @samp{r} @tab Red
 ## @item @samp{g} @tab Green
 ## @item @samp{b} @tab Blue
 ## @item @samp{m} @tab Magenta
 ## @item @samp{c} @tab Cyan
 ## @item @samp{w} @tab White
 ## @end multitable
 ##
-## @item @qcode{";key;"}
-## Here @qcode{"key"} is the label to use for the plot legend.
+## @item @qcode{";displayname;"}
+## Here @qcode{"displayname"} is the label to use for the plot legend.
 ## @end table
 ##
-## The @var{fmt} argument may also be used to assign legend keys.
+## The @var{fmt} argument may also be used to assign legend labels.
 ## To do so, include the desired label between semicolons after the
 ## formatting sequence described above, e.g., @qcode{"+b;Key Title;"}.
 ## Note that the last semicolon is required and Octave will generate
 ## an error if it is left out.
 ##
 ## Here are some plot examples:
 ##
 ## @example
